src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -2792,4 +2792,17 @@ public GHTagObject createTag(String tag, String message, String object, String t
                 .fetch(GHTagObject.class)
                 .wrap(this);
     }
+
+    /**
+     * Repopulates this object.
+     *
+     * @throws java.io.IOException
+     *             The IO exception
+     */
+    public void refresh() throws IOException {
```

Comment:
For now let's make this package private and call it `populate`:
```suggestion
    void populate() throws IOException {
```